pkgs/development/python-modules/pyxbe/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pyxbe";
+  version = "0.0.2";
```

Comment:
You probably want to get a more recent "version": bug fixes were added to `master` without tags on GH (see https://github.com/mborgerson/pyxbe/commits/master).
Here is what I have: https://github.com/angr/nixpkgs/blob/master/pkgs/python-modules/pyxbe/default.nix .

As I wrote this comment, I realised that a `0.0.3` has been published "recently" on https://pypi.org/project/pyxbe/ , but AFAICT, without the tests.

Reply:
There is an open PR for tagged releases: https://github.com/mborgerson/pyxbe/issues/10

I will go with a commit for now.